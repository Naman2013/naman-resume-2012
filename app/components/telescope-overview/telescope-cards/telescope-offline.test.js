import React from 'react';
import renderer from 'react-test-renderer';
import TelescopeOffline from './telescope-offline';

it('renders correctly', () => {
  const tree = renderer.create(<TelescopeOffline />).toJSON();
  expect(tree).toMatchSnapshot();
});
