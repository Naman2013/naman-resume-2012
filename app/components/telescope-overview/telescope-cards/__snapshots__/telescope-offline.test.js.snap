// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="image-viewer"
  >
    <h4
      className="title"
    >
      Offline
    </h4>
    <div
      className="telescope-image"
    >
      <img
        alt="This telescope is currently offline"
        height="245"
        src={undefined}
        width="245"
      />
    </div>
    <p
      className="telescope-status"
    >
      
    </p>
  </div>
</div>
`;
